### ISSUE TITLE
Not support HTML Input File

### ISSUE THREAD

**Issue by heavenwing** [SKLn-Rad/Xam.Plugin.Webview#66]
I write a html that contains Input element with type=file, and file upload button is not working.

**Comment by heavenwing** [comment]
I found a solution : https://forums.xamarin.com/discussion/62284/input-type-file-doesnt-work-on-xamarin-forms-webview-androidhome.firefoxchina.cn
I verified it's OK, I will send a pull request.

**Comment by SKLn-Rad** [comment]
Yes if you want! That would be a great help. I current do the same checks for "mailto" as well so potentially you could do a PR next to that?

Thanks a lot!

**Comment by Joshin95** [comment]
is this bug solved? because with the last version(2.1.3) still I'm not able to use the input button.

Thanks,

**Comment by Kauppi** [comment]
Hi @heavenwing, did you find a way to solve the input file browsing support? I have tried out some solutions without any luck.
